frontend/src/components/search/SearchResultsTable.tsx

```diff
@@ -169,11 +157,6 @@ export const SearchResultsTable = async ({
     return <FilterSearchNoResults useHeading={true} />;
   }
 
-  const savedOpportunities = await fetchSavedOpportunities();
```

Comment:
do we lose out on any real efficiency from making these requests server side here? Probably not.